crates/orderbook/src/database/orders.rs

```diff
@@ -34,6 +34,7 @@ use std::convert::TryInto;
 pub trait OrderStoring: Send + Sync {
     async fn insert_order(&self, order: &Order, quote: Option<Quote>)
         -> Result<(), InsertionError>;
+    async fn cancel_orders(&self, order_uids: Vec<OrderUid>, now: DateTime<Utc>) -> Result<()>;
     async fn cancel_order(&self, order_uid: &OrderUid, now: DateTime<Utc>) -> Result<()>;
```

Comment:
nit: Is it intended that the single cancellation takes a reference whereas the multi order cancellation takes an owned value?
Looks like taking a `&[OrderUid]` would be fine here given the fact that `OrderUid` implements `Copy`.

Reply:
> Looks like taking a &[OrderUid] would be fine here given the fact that OrderUid implements Copy.

We need to move the `Vec` into a future, which is why we use it instead of a slice.